### DISCUSSION TITLE
What are "Notes" for?

### DISCUSSION THREAD

**Discussion by luciochavez** [senshu/Sozi#548] [Q&A] [answered]
When i write a note in a frame it doesn't show anywhere in presentation. Is it just for editing purposes? or Does i have to do something to display it in presentation?

**Comment by luciochavez** [comment, marked as answer]
Notes can only be viewed from the presentation file:
NAME-presenter.sozi.html

**Reply by ejvindh** [reply]
"Notes" are for the presenter only. The idea being that you show the "ordinary" html on a screen that is visible for the audience, and have the "notes" on the screen that is only visible for the presenter. They will then change in parallel. Can be useful e.g. when you want to supplement your memory with ideas about what to say while showing the slides.
